docs/beta/package.json

```diff
@@ -32,7 +31,7 @@
     "web-vitals": "^0.2.4"
   },
   "scripts": {
-    "predeploy": "npm run build",
+    "predeploy": "yarn run build",
```

Comment:
`predeploy` is not a lifecycle script and doesn't seem to be used anywhere :thinking: maybe we can just get rid of this?

Reply:
Maybe. Trying to keep this focused to yarn/npm though